x-pack/qa/smoke-test-plugins-ssl/build.gradle

```diff
@@ -29,132 +30,13 @@ project.sourceSets.test.output.dir(outputDir, builtBy: copyXPackPluginProps)
 // needed to be consistent with ssl host checking
 Object san = new SanEvaluator()
```

Comment:
The certs that are pre-created, what do they have for a SAN value? We used to dynamically generate this on the fly so that we could still use hostname verification

Reply:
> We used to dynamically generate this on the fly so that we could still use hostname verification

Yes, I know. It's a little more complicated now - we touched upon generation and potential issues with Alpar also yesterday: https://github.com/elastic/elasticsearch/pull/31989#discussion_r201990240 


> The certs that are pre-created, what do they have for a SAN value? 

```
 X509v3 Subject Alternative Name: 
                DNS:localhost, DNS:localhost.localdomain, DNS:localhost4, DNS:localhost4.localdomain4, DNS:localhost6, DNS:localhost6.localdomain6, IP Address:127.0.0.1, IP Address:0:0:0:0:0:0:0:1
```

I think it will suffice, once CI reaches that we'll get a verification. 
I 'd need to remove the SanEvaluator , as it is not used anymore.

Reply:
that looks close to what we used to use, lets go with it and see if we have issues